vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/EmailFieldPageIT.java

```diff
@@ -126,20 +125,6 @@ public void assertClearValue() {
         Assert.assertEquals("Old value: 'foo'. New value: ''.", value);
     }
 
-    @Test
-    public void assertInvalidValue() {
```

Comment:
**note:** Already covered by this test:

https://github.com/vaadin/flow-components/blob/0dc6fc8aba424dee207dd757ea5a140a2e3b6ddb/vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/validation/EmailFieldBasicValidationIT.java#L118-L127